stdlib/3/typing.pyi

```diff
@@ -289,20 +311,23 @@ class ValuesView(MappingView, Iterable[_VT_co], Generic[_VT_co]):
     def __contains__(self, o: object) -> bool: ...
     def __iter__(self) -> Iterator[_VT_co]: ...
 
-class ContextManager(Generic[_T_co]):
+@runtime
+class ContextManager(Protocol[_T_co]):
     def __enter__(self) -> _T_co: ...
     def __exit__(self, exc_type: Optional[Type[BaseException]],
                  exc_value: Optional[BaseException],
                  traceback: Optional[TracebackType]) -> Optional[bool]: ...
 
 if sys.version_info >= (3, 5):
-    class AsyncContextManager(Generic[_T_co]):
+    @runtime
+    class AsyncContextManager(Protocol[_T_co]):
         def __aenter__(self) -> Awaitable[_T_co]: ...
         def __aexit__(self, exc_type: Optional[Type[BaseException]],
                       exc_value: Optional[BaseException],
                       traceback: Optional[TracebackType]) -> Awaitable[Optional[bool]]: ...
 
-class Mapping(_Collection[_KT], Generic[_KT, _VT_co]):
+@runtime  # type: ignore
```

Comment:
Actually, it looks like is is not necessary anymore. The point is that ``Mapping`` is not really covariant in value (see signature of ``get``) and mypy is deliberately very strict about such situations with protocols, however it looks like treatment of unions has changed since this was added. I will remove it just in case.